### DISCUSSION TITLE
If many individuals are using identical devices in one building, they count as one visitor.

### DISCUSSION THREAD

**Discussion by whscienceteacher** [plausible/analytics#1525] [Anything Else]
The current system for identifying unique visitors doesn't work in a setting like a school where the students are given centrally administered devices and share an IP address.

According to the documentation, the unique identifier is:

`hash(daily_salt + website_domain + ip_address + user_agent)`

If many devices are connected to one wifi network they may share an IP-address. If they are all the same device and centrally updated, they likely have the same user agent. 

I don't have a solution that preserves the privacy focus of plausible but if anyone else has an idea I would appreciate it. My website is mostly used in schools, many of which are using class sets of chromebooks so I can't see how many people are using the site at each location.

**Comment by metmarkosaric** [comment]
hi @whscienceteacher! yeah, that's correct. if multiple of your visitors have the exact same IP address and use exact same device then our unique visitors number would see them as the same visitor. we don't have a good solution for that at the moment unfortunately. introducing a first party cookie could work for this use case but it's not something we offer at this stage.

**Reply by whscienceteacher** [reply]
Users with the same IP and device still register separate views. Would it be possible to give the option to see the top pages, location, and devices sections of the website by pageviews instead of visitors?

**Reply by metmarkosaric** [reply]
at the moment it's not possible to replace visitors with pageviews in our UI. if you click on the "details" view in the Top Pages report you can see the pageviews there already. if you filter by any page or any location or any device, you can see the total pageview number in the top graph too. we have this data in our [stats API](https://plausible.io/docs/stats-api) too which may be a more convenient way to pull them. you can also get it so you can sort it etc using our [CSV export](https://plausible.io/docs/export-stats) functionality

**Reply by benjick** [reply]
No workaround for this yet?

**Reply by mgussekloo** [reply]
Sorry for upvoting an old topic, but I would like this as well. Perhaps a method to trigger a "the current visit is over, a new unique visit starts" event? Any tips or tricks welcome.
